QA v1.7.0-rc32

EZheln opened the pull request:
Updated tests according to requirements
- updated headless test running
- removed tests with @oldJobWizard tag
- updated frontendSpec file
- added data updating for project-summaries request (workflows and scheduled)
- added filtering for workflows and scheduled after redirection from Projects page

Taras-Hlukhovetskyi commented on tests/mockServer/mock.js:
```diff
@@ -1242,30 +1245,40 @@ function deleteProjectsFeatureVectors (req, res) {
+      runs: collectedMonitoringPipelines, 
       total_size: collectedMonitoringPipelines.length,
-      next_page_token: null
+      next_page_token: null 
```
redundant space after null

Commit:
Updated tests according to requirements
- updated headless test running
- removed tests with @oldJobWizard tag
- updated frontendSpec file
- added data updating for project-summaries request (workflows and scheduled)
- added filtering for workflows and scheduled after redirection from Projects page
